### ISSUE TITLE
3.0 - Add $attr option for datetime parts select lists.

### ISSUE THREAD

**Issue by commercial-hippie** [cakephp/cakephp#5829]
The following functions don't support adding HTML attributes:

``` php
$this->Form->year()
$this->Form->month()
$this->Form->day()
$this->Form->hour()
$this->Form->minute()
$this->Form->second()
$this->Form->meridian()
```

I will try do this one myself if its a good idea? :smile:

**Comment by markstory** [comment]
I think it makes sense to support HTML attributes. How do you feel about adding an `$attributes` parameter? This would make these methods work like FormHelper::select() does.

**Comment by dereuromark** [comment]
:+1:

**Comment by andrecavallari** [comment]
Following... I tried to modify by my own, but didnt success... =(

**Comment by mhagenbruch** [comment]
Does anybody know when this feature will be implemented in core?

**Comment by ADmad** [comment]
Fixed in #5861
